libs/elements/layout/carousel/src/lib/comoponents/project-carousel/project-carousel.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-website-project-carousel',
+  templateUrl: './project-carousel.component.html',
+  styleUrls: ['./project-carousel.component.scss'],
+})
+
+export class ProjectCarouselComponent {
+  
+
+  public imgCollection: string[] = [
+   'https://www.theindiaforum.in/sites/default/files/field/image/2022/06/21/ramkumar-radhakrishnan-wikimedia-1622193304-1622193304.jpeg',  
+   'https://static.toiimg.com/photo/msid-92790134/92790134.jpg',
```

Comment:
Should be an Input. There was no need to change this. The only change that was needed here is the type of data the input should be.